subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/tasks/rules/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Implementations for Jacoco code coverage rules.
```

Comment:
I think it would feel kind of weird to link to the documentation page of the Ant task. That documentation page also doesn't have any anchors so users would have to scroll down to even find what they are looking for. It would probably confuse more than help.